src/vivarium_gates_mncnh/components/mortality.py

```diff
@@ -256,14 +256,17 @@ def on_time_step(self, event: Event) -> None:
         ]:
             return
 
+        #breakpoint()
         pop = self.population_view.get(event.index)
         alive_idx = pop.index[pop[COLUMNS.CHILD_ALIVE] == "alive"]
         mortality_rates = self.death_in_age_group(alive_idx)
+        mortality_rates = self.death_in_age_group.source(alive_idx)  # I can't figure out where this pipeline is being modified, but I think there is something wrong with the modification
+
         # Convert to rates to probability
         if self._sim_step_name() == SIMULATION_EVENT_NAMES.EARLY_NEONATAL_MORTALITY:
-            duration = 7 / 365.0
+            duration = .07 / 365.0
```

Comment:
Perhaps the large step size is causing problems?

Reply:
Don't we need it to be 7 days to convert it to a probability correctly?

Reply:
I no longer think this is the problem.